package.json

```diff
@@ -98,22 +98,25 @@
     "@mui/lab": "^5.0.0-alpha.109",
     "@mui/material": "^5.10.15",
     "@mui/x-date-pickers": "^5.0.9",
-    "@origyn-sa/mintjs": "^1.0.0-alpha.1",
-    "@origyn-sa/origyn-art-ui": "1.2.1",
+    "@origyn-sa/mintjs": "^1.0.0-alpha.2",
+    "@origyn-sa/origyn-art-ui": "1.0.52",
```

Comment:
Ok, so you did merge wrong if you overrides existing updates with your code... Check it properly, this cant be merged at the moment